Makefile

```diff
@@ -0,0 +1,21 @@
+NAME=tendrl-common
+VERSION=0.0.1
+
+all: srpm
+
+clean:
+	rm -rf dist/
+	rm -rf $(NAME)-$(VERSION).tar.gz
+	rm -rf $(NAME)-$(VERSION)-1.el7.src.rpm
+
+dist:
+	python setup.py sdist \
+	  && mv dist/$(NAME)-$(VERSION).tar.gz .
+
+srpm: dist
+	fedpkg --dist epel7 srpm
+
+rpm: dist
+	mock -r epel-7-x86_64 rebuild $(NAME)-$(VERSION)-1.el7.src.rpm --resultdir=.
```

Comment:
I recommend adding `--define "dist .el7"` to this mock invocation here, so that it does not use [CentOS's default](http://bugs.centos.org/view.php?id=7416) of `.el7.centos`